modules/apps/portal-security/portal-security-ldap-impl/src/main/java/com/liferay/portal/security/ldap/internal/SafePortalLDAPImpl.java

```diff
@@ -495,6 +495,17 @@ public Binding getUser(
 			String emailAddress, boolean checkOriginalEmail)
 		throws Exception {
 
+		return getUser(
+			ldapServerId, companyId, screenName, emailAddress, false, true);
```

Comment:
I think you should pass `checkOriginalEmail` instead of `false` but if you follow my later comment it won't matter.